webapp-django/crashstats/crashstats/views.py

```diff
@@ -908,8 +908,23 @@ def append_row_blob(blob, labels):
     return response
 
 
+def crashes_per_user_redirect(request):
+    """Redirect to the new named URL which is crashes_per_day.
+    The reason for NOT putting this directly in the urls.py definitions
+    is because then we can't carry the query string.
+    """
+    url = reverse('crashstats:crashes_per_day')
+    if request.META.get('QUERY_STRING'):
+        url += '?{}'.format(request.META['QUERY_STRING'])
+    # At some point later in 2016, we can confidently change this to:
+    # `permanent=True` when we know the redirect is working correctly.
+    # In the transition time, it's safer to use a temporary redirect
+    # since permanent redirects tend to get very stuck in the brower.
+    return redirect(url, permanent=not settings.DEBUG)
```

Comment:
I find it weird to have this redirect as a view instead of directly in the ``urls.py`` file. I'm not sure the flexibility in dev is worth the added complexity.